apps/kstream/README.md

```diff
@@ -348,11 +348,11 @@ $ helm install --name kstream ./helm
 $ kubectl logs <pod name where the application runs>
 ```
 
-Same technique can be used to deploy all the sample applications in Kubernetes.
+Same technique can be used to deploy all the sample applications in Kubernetes or OpenShift.
 
 ### Using the traefik Ingress Controller
 
-`kstream` application publishes a REST end point and allows users to query on various application state information. In Kubernetes this can be done in a way so that the user gets one fixed end point irrespective of the ways the underlying PODs are distributed across the cluster. Even in case of PODs getting restarted on a different node, it would be great to still have the fixed end point. We use the traefik ingress controller for this.
+`kstream` application publishes a REST end point and allows users to query on various application state information. In Kubernetes or OpenShift this can be done in a way so that the user gets one fixed end point irrespective of the ways the underlying PODs are distributed across the cluster. Even in case of PODs getting restarted on a different node, it would be great to still have the fixed end point. We use the traefik ingress controller for this.
 
 > Ensure that the traefik ingress controller is installed in the cluster. Check that the dashboard is accessible through some end points like http://<public agent IP>:9901/dashboard/
```

Comment:
I think we want to remove this and write something like:
To expose service on kubernetes use Ingress (on vanila kubernetes) or Route (on OpenShift)

Reply:
Okay. I just pushed a small change to the text here. See what you think.

Reply:
Looks good, but I would probably remove traefik completely. It was a hack that we did for k8 on DC/OS